ledger-core/v2/conveyor/conveyor.go

```diff
@@ -107,6 +107,10 @@ func (p *PulseConveyor) AddDependency(v interface{}) {
 	p.slotMachine.AddDependency(v)
 }
 
+func (p *PulseConveyor) AddDependencyByVar(v interface{}) {
```

Comment:
I don't understand picked name, for example in the following code name doesn't explain difference between two calls to different variants of AddDependency:  
```
	slotMachine.AddDependency(&pulseSlot)

	var objectCatalog object.Catalog = object.NewLocalCatalog()
	slotMachine.AddDependencyByVar(&objectCatalog)
```